<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableExperiments">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="horizontalWidget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="bttnAdd">
           <property name="text">
            <string>▼</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttnRemove">
           <property name="text">
            <string>▲</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableComparison">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="verticalWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="bttnLDH">
             <property name="text">
              <string>LDH</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bttnMTT">
             <property name="text">
              <string>MTT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboNormalize"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboSignificance"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft">
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cell Line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboCellLines">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Graph Title:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtGraphTitle">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
